source/style-guide/style/toc-labels.txt

```diff
@@ -0,0 +1,117 @@
+  keywords to improve the page's discoverability in search results. To 
+  learn more, see :ref:`seo-guidelines`.
+
+- Create succinct, meaningful, descriptive page titles, and place the 
+  most important words first. Page titles should have a maximum of 70 
+  characters and provide a quick overview of the content. Search 
+  results or browser tabs might truncate extra long titles. Avoid 
+  excessive or irrelevant words (keyword stuffing).
+
+- While we should craft concise titles, include enough information to 
+  confirm to the user that they have landed in the right place. A page 
+  title can provide a bit more detail than a TOC label.
+
+- Ensure that each title and heading is unique. Identical titles, even 
+  between documentation sets, might compete in search results.
+
+- Don't include "MongoDB" in a title unless the page is a product 
+  landing page.
+
+- Aim to attract users and encourage them to click on the link.
+
+- Create page titles that are readable and effective on various 
+  devices, including mobile. This involves considering character limits 
+  and readability on smaller screens.
+
+- Use a consistent grammatical structure for page titles. Use a 
+  consistent style and tone across all page titles to maintain a 
+  unified and professional appearance. For example, use verbs to 
+  indicate procedural content.
+
+To learn more, see :ref:`titles-and-headings`.
+
+TOC Labels
+----------
+
+- While the automatic labels make it easier for us to 
+  update the TOC as the content evolves, long titles interfere with 
+  efficient navigation. So, we recommend that you specify TOC labels to 
+  enhance the document's accessibility and usability.
+
+- TOC labels primarily represent the documentation's structure. They 
+  outline the hierarchy of sections and help users navigate through the 
+  content.
+
+- Craft clear and descriptive TOC labels that convey the content of 
+  each section or page accurately. Users should understand the context 
+  from the TOC label and its placement in the overall TOC hierarchy.
+
+- Avoid line-wrapping. Let the placement of the TOC label in the 
+  overall TOC hierarchy provide context. Avoid redundancy and 
+  repetition. The page title itself can provide a bit more detail and 
+  confirm to the user that they have landed in the right place. To this 
+  end, you can use industry standard abbreviations and the ampersand 
+  when needed.
+
+  .. important::
+
+     In favor of brevity, TOC labels do allow for the the ampersand ( & ) symbol.
+
+- Don't stray so far from the original page title that you disorient 
+  the user.
+
+- TOC labels should follow a logical order that reflects the 
+  documentation's flow. You might include numbering or an order that 
+  makes sense for the documentation's purpose.
+
+- TOC labels primarily aid user navigation. Users should be able to 
+  quickly locate and go to specific sections of interest.
```

Comment:
```suggestion

```
Think this point could be removed or added on to the second bullet, as they say similar things: "TOC labels primarily represent the documentation's structure. They outline the hierarchy of sections and help users navigate through the 
content."